src/main/java/net/coderbot/iris/pipeline/DeferredWorldRenderingPipeline.java

```diff
@@ -194,13 +194,18 @@ public DeferredWorldRenderingPipeline(ProgramSet programs) {
 
 		this.centerDepthSampler = new CenterDepthSampler(renderTargets, updateNotifier);
 
-		flippedBeforeTranslucent = flipper.snapshot();
+		flippedAfterPrepare = flipper.snapshot();
```

Comment:
You're taking the flippedAfterPrepare snapshot before the prepare renderer has been initialized? That doesn't seem right.

Reply:
Fixed.